src/main/java/com/solvd/laba/services/UserService.java

```diff
@@ -0,0 +1,29 @@
+package com.solvd.laba.services;
+
+import org.openqa.selenium.WebDriver;
+
+import com.solvd.laba.config.EnvironmentConfig;
+import com.solvd.laba.models.User;
+import com.solvd.laba.pages.HomePage;
+import com.solvd.laba.pages.LoginPage;
+
+public class UserService {
```

Comment:
functionality described here is actually about logging in a user. it would be better to rename it to something like LoginService to reflect its purpose more clearly.